cli/src/cmd/install.rs

```diff
@@ -124,7 +127,8 @@ fn install_as_folder(dep: &Dependency, libs: &Path) -> eyre::Result<()> {
 fn install_as_submodule(dep: &Dependency, libs: &Path, no_commit: bool) -> eyre::Result<()> {
     // install the dep
     let output = Command::new("git")
-        .args(&["submodule", "add", &dep.url, &dep.name])
+        // --quiet prevents git from sending info messages to stderr
+        .args(&["submodule", "add", "--quiet", &dep.url, &dep.name])
```

Comment:
same

Reply:
In this case the correct position is apparently after `submodule` so I'll update this for consistency.